README.md

```diff
@@ -1,30 +1,115 @@
+[![Coverage Status][codecov-image]][codecov-url] [![Code style:
+ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+
+**Snapcraft** is the command-line build tool for packaging and distributing software and
+apps in the snap container format. It solves the problems of dependency management and
+architecture support by bundling all of a software's libraries into the container
+itself, and gives developers a way to package any app, program, toolkit, or library for
+all major Linux distributions and IoT devices.
+
+## Basic usage
+
+A snap's build configuration is stored in simple language as a project file called
+`snapcraft.yaml`, making it easy to add as a new package format to your existing code
+base.
+
+From the root of the code base of any software project, Snapcraft creates a minimal
+`snapcraft.yaml` with:
+
+```bash
+snapcraft init
+```
+
+After you add all your project's build and runtime details to the project file, bundle
+your project into a snap with:
+
+```bash
+snapcraft
+```
+
+Your project can be registered on public and private app stores, including the Snap
+Store:
+
+```bash
+snapcraft register
+```
+
+Snap versions and revisions, including parallel releases, are published to the store
+with:
+
+```bash
+snapcraft upload
+```
+
+If you're interested in learning more about the Snapcraft commands and how to compose a
+project file, try [creating your first
+snap](https://snapcraft.io/docs/create-a-new-snap).
 
-Snaps are containerised software packages that are simple to create and
-install. They auto-update and are safe to run. And because they bundle their
-dependencies, they work on all major Linux systems without modification.
+## Installation
 
-[Build your first snap](https://docs.snapcraft.io/build-snaps/languages) or learn more about how [Snapcraft can help you](https://snapcraft.io).
+Snapcraft is available on all major Linux distributions, Windows, and macOS.
 
-## Get support
+Snapcraft itself has first-class support as a snap. On snap-ready systems, you can install it on the command line with:
 
-We’re here to help. Ask your questions at the [Snapcraft Forum](https://forum.snapcraft.io). Report bugs on [Launchpad](https://bugs.launchpad.net/snapcraft/+filebug).
+```bash
+sudo snap install snapcraft --classic
+```
 
-Learn about the latest features by following Snapcraft on
-[Twitter](https://twitter.com/snapcraftio) or
-[Facebook](https://www.facebook.com/snapcraftio).
+For complete installation, you need an additional Linux container tool. Snapcraft can
+also be installed as a traditional package on many popular Linux repositories. For help
+with both, we cover how to [set up
+Snapcraft](https://canonical-snapcraft.readthedocs-hosted.com/en/stable/howto/set-up-snapcraft)
```

Comment:
Idiomatic example of a line we can't keep within an 88 char limit. :upside_down_face: